plugins/woocommerce-admin/client/customize-store/assembler-hub/hooks/use-popover-handler.ts

```diff
@@ -47,7 +47,7 @@ export const usePopoverHandler = () => {
 		hoveredBlockClientId: string | null;
 	} ) => {
 		const iframe = window.document.querySelector(
-			'.woocommerce-customize-store-assembler > iframe[name="editor-canvas"]'
+			'.woocommerce-customize-store-assembler > .block-editor-iframe__container iframe[name="editor-canvas"]'
```

Comment:
The iframe is no longer an immediate child of `.woocommerce-customize-store-assembler` so selectors were not detecting the iframe. I tried to make it specific enough so that it would continue picking up the iframe with a level of specificity which would not result in getting the wrong iframe since there are multiple.

Reply:
It makes sense! Thanks! 🙇